tests/test_mutualauth.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+import unittest
+
+import falcon
+from falcon_mutualauth.mutualauth import Authorize
+
+
+class FakeRequest(object):
+
+    def __init__(self, role):
+        self.__role = role
+        self.env = {'REMOTE_ADDR': 'http://test.test'}
+        self.user_agent = 'pytest'
+
+    def get_header(self, header):
+        assert header == 'X-User-Roles'
+        return self.__role
+
+
+class FakeResource(object):
+
+    def __init__(self, authorized_for):
+        self.authorized_for = authorized_for
+
+
+class TestAuthorize(unittest.TestCase):
+
+    def setUp(self):
+        self._auth = Authorize('admin', 'principle')
+        self._resource = self._create_fake_resource()
+
+    def _create_fake_resource(self):
+        return None
+
+    def test_log_in_as_admin(self):
+        self._auth.process_resource(FakeRequest('admin'), None, self._resource)
+
+    def test_log_in_as_principle(self):
+        self._auth.process_resource(
+            FakeRequest('principle'), None, self._resource)
```

Comment:
I see that https://github.com/RackSec/falcon-mutualauth/blob/master/falcon_mutualauth/mutualauth.py#L78 splits on a comma---maybe we could exercise the set path by using multiple roles in one of these tests?